src/utils/validate.js

```diff
@@ -40,9 +40,11 @@ function getPropertyName(error: Object): string {
 
 function getErrorArgs(error) {
   switch (error.keyword) {
+    case 'maxItems':
```

Comment:
WDYT about add `maxProperties`/`minProperties` to objects and maybe `exclusiveMaximum`/`exclusiveMinimum` to keywords or numbers? Check this links below: 

https://ajv.js.org/keywords.html#maxproperties--minproperties
https://ajv.js.org/keywords.html#maximum--minimum-and-exclusivemaximum--exclusiveminimum

Reply:
Sure, but I'll do that in a different PR.

Reply:
Done in #16 and #17

Reply:
Done in #16 and #17